### PR TITLE
Update Link in Connect Scroll on Home Page of celo.org

### LINKED ISSUE #176: Update Link in Connect Scroll on Home Page of celo.org
Screen Shot 2021-04-27 at 5 10 14 PM

Change "Join an Event" to "Chat with the Community"

### PR DESCRIPTION
**Pull request by Henrynunez112**
**Description**

*A few sentences describing the overall effects and goals of the pull request's commits.*
- Updated the link in the Connect Scroll on the home page of the Celo.org website, on the secondary link change the from "Join an Event" to "Chat with the Community". Lastly, removed the "Apply for Fellowship" link.

*What is the current behavior, and what is the updated/expected behavior with this PR?*
Screen Shot 2021-04-29 at 1 23 05 PM

Screen Shot 2021-04-29 at 1 23 21 PM

*Removed the Apply for Fellowship link*
Screen Shot 2021-04-29 at 2 03 59 PM

*Tested*
176
Screen Shot 2021-04-29 at 1 30 36 PM

177
Screen Shot 2021-04-29 at 2 07 48 PM

Fixes #176, #177

### CODE REVIEW

File: src/home/Involvement.tsx [hunk 1]
```diff
@@ -178,13 +177,13 @@ function Content({ path }) {
               style={styles.primary}
               href={LINKS[path].primary}
             />
-            <Button
+            {path === 5 ? null : <Button
```
Comment (aaronmgdr) [inline src/home/Involvement.tsx:180]: the path === 5 here is very brittle 

a more robust way is to see if there is a secondary link  

` !LINKS[path].secondary ? null : <Button`

Comment (Henrynunez112) [reply src/home/Involvement.tsx:180]: 🔥

File: public/static/locales/en/home.json [hunk 2]
```diff
@@ -88,14 +88,13 @@
         "name": "Connect",
         "title": "Join the Celo movement",
         "primary": "Become an Ambassador",
-        "secondary": "Join an Event",
+        "secondary": "Chat with the Community",
```
Comment (aaronmgdr) [inline public/static/locales/en/home.json:91]: the link for this should change to i think. to chat.celo.org

**Review by aaronmgdr — APPROVED** [review]
🚢

**Pull request merged**